changes.d/686.feat.md

```diff
@@ -0,0 +1 @@
+657.feat.md
```

Comment:
Need a better changelog entry!

Reply:
Ah right, this doesn't need a changelog, just delete the file.

Reply:
Happy to merge as is? It's harmless enough and IMO mildly convenient to link to both PRs

Reply:
Yes, please take approval with comments as agreement to merge with suggestions.